cookbooks/convection-box.prm

```diff
@@ -101,7 +101,9 @@ end
 
 subsection Formulation
   set Formulation = custom
-  set Temperature density approximation = adiabatic
+  set Mass conservation    = incompressible
+  set Temperature equation = real density #reference density profile
```

Comment:
I can go with whatever formulation you think is right (though I think that our formulation is better than all of these approximations, and so would just use it).

I'm not fond of comments that (i) do not actually explain anything, (ii) reflect the *history* of a piece of a code rather than what is happening *now*. I just don't know what you want to express with the comment, and I suspect that a reader of that file wouldn't know either.